### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by drwpow**
## Reason for change

This README was being linked to from [our partner docs](https://ui.sandbox.manifold.co) but was mostly inaccurate / incomplete.

It also shipped internal development notes as part of the consumer README.

## Testing

Spellcheck & copywriting feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,95 @@
...
+### HTML / JS
 
 ```html
-<manifold-oauth oauth-url="[your-oauth-url]"></manifold-oauth>
+<body>
+  <manifold-auth></manifold-auth>
+  <script src="https://unpkg.com/@manifoldco/shadowcat/dist/manifold.js"></script>
+  <script>
+    document
+      .querySelector("manifold-auth")
```
Comment (sslotsky) [inline README.md]: I think this is a typo, should be `.querySelector("manifold-oauth")` right?

File: src/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Shadowcat dev
+
+## Dev Setup
+
+1. `npm install` to install dependencies.
+1. `npm start` to start the development server.
+1. Go to `localhost:3333`. The development server will display a blank page.
+1. Open the console. There will be a message/token logged.
+
+## Publish to npm
```
Comment (sslotsky) [inline src/README.md:10]: I believe you actually fixed the autopublish flow for this repo, right? Should we update this section?

File: README.md (hunk 1 above)
Comment (drwpow) [reply README.md]: Oh you’re right. Will fix.

File: README.md [hunk 3]
```diff
@@ -1,47 +1,95 @@
...
-  console.log(e.detail); // received token
-  // do stuff...
-});
-```
+In any setup, you can use a CDN for Shadowcat
 
-## Dev Setup
+```
+https://js.cdn.manifold.co/@manifoldco/shadowcat/       # latest (beware of breaking changes!)
+https://js.cdn.manifold.co/@manifoldco/shadowcat/0.2.0/ # specific version
```
Comment (domenicrosati) [inline README.md]:
```suggestion
https://js.cdn.manifold.co/@manifoldco/shadowcat@0.2.0/ # specific version
```

**Review by domenicrosati — CHANGES_REQUESTED** [review]

**Review by domenicrosati — APPROVED** [review]
ship

**Pull request merged**